pkg/i18n/i18n.go

```diff
@@ -37,7 +32,10 @@ func NewLang(code string) LanguageCode {
 	} else if strings.Contains(code, "ja") {
 		return JA
 	}
-	return ZH
+	if i18nCode, ok := i18nCodeMap[code]; ok {
+		return i18nCode
+	}
+	return EN
 }
 
 func T(s string) string {
```

Comment:
There do not appear to be any irregularities, issues, or areas of optimization related to this Go code snippet from 2021-09-01. The provided functions seem correct, and they perform operations that involve setting up configuration files, configuring languages into locales, loading localized strings, adding domains to locales when necessary, and determining appropriate language codes based on those configurations.

If you have specific questions about the functionality or performance of these methods, please let me know!